webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/index.js

```diff
@@ -0,0 +1,141 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Card,
+  CardHeader,
+  CardTitle,
+  CardBody,
+  CardActions,
+  ClipboardCopy,
+  Divider,
+} from '@patternfly/react-core';
+import { UserIcon } from '@patternfly/react-icons';
+import { foremanUrl } from '../../../common/helpers';
+import PowerStatusDropdown from './PowerStatusDropdown';
+import { translate as __ } from '../../../common/I18n';
+import SkeletonLoader from '../../common/SkeletonLoader';
+import { STATUS } from '../../../constants';
+
+const DetailsCard = ({
+  id: hostID,
+  ip,
+  ip6,
+  mac,
+  comment,
+  owner_id: ownerID,
+  owner_name: ownerName,
+  hostgroup_name: hostgroupName,
+  hostgroup_id: hostgroupID,
+  permissions: { power_hosts: canUsePower },
+  status,
+}) => (
+  <Card isHoverable>
+    <CardHeader>
+      <CardTitle>{__('Details')}</CardTitle>
+      <CardActions>
+        <PowerStatusDropdown hostID={hostID} hasPowerPermission={canUsePower} />
+      </CardActions>
+    </CardHeader>
+    <CardBody>
+      <DescriptionList
+        isAutoColumnWidths
+        columnModifier={{
+          default: '2Col',
+        }}
+      >
+        <DescriptionListGroup>
+          <DescriptionListTerm>
+            {ip6 ? __('IP6 Address') : __('IP Address')}
```

Comment:
The question is whether people don't prefer to see one of them in that case, which was the logic behing this decision of a bigger group. We can display both and hide the field it it's note present, which eats more space. I think we're dropping one of the UUID though, so perhaps that would be OK.

OTOH people should also have the complete list of interfaces with all addresses exposed in some way on the host detail page. So in this overview, I'm not sure we need to display both. If someone deployes IPv6, they probably want to use IPv6 mainly. Yes, there are cases you want to actively use dual stack, but in such case I'd say FQDN is more important.

No strong argument, @amirfefer could you please modify this so we display both (each only if the value is present)?

Reply:
As a user, I'd be highly surprised if you hide one. A host can be:
* IPv4 enabled
* IPv6 enabled
* IPv4 & IPv6 enabled (dual stack)
Only showing one means you can't distinguish between IPv6-only and dual stack. It also makes it hard to figure out the IPv4 address (which is typically still used a lot in dual stack deployments). I strongly insist you show both.

Less insistent, but in most places you write IPv6, not IP6. It would also be consistent with pretty much all other labels in the application.